src/main/scala/mesosphere/marathon/core/election/impl/ElectionServiceBase.scala

```diff
@@ -15,7 +15,7 @@ import scala.concurrent.Future
 import scala.util.control.{ ControlThrowable, NonFatal }
 
 private[impl] object ElectionServiceBase {
-  protected type Abdicator = /* error: */ Boolean => Unit
+  type Abdicator = /* error: */ Boolean => Unit
```

Comment:
If we go with the approach I suggested then it should not be necessary to expose this